src/gluonts/model/forecast.py

```diff
@@ -283,99 +284,73 @@ def __getitem__(self, name):
 
     def plot(
         self,
-        prediction_intervals=(50.0, 90.0),
-        show_mean=False,
-        color="b",
-        label=None,
-        output_file=None,
-        *args,
-        **kwargs,
+        *,
+        intervals=(0.5, 0.9),
+        ax=None,
+        color=None,
+        name=None,
+        show_label=False,
     ):
         """
-        Plots the median of the forecast as well as prediction interval bounds
-        (requires matplotlib and pandas).
+        Plot median forecast and prediction intervals using ``matplotlib``.
 
-        Parameters
-        ----------
-        prediction_intervals : float or list of floats in [0, 100]
-            Prediction interval size(s). If a list, it will stack the error
-            plots for each prediction interval. Only relevant for error styles
-            with "ci" in the name.
-        show_mean : boolean
-            Whether to also show the mean of the forecast.
-        color : matplotlib color name or dictionary
-            The color used for plotting the forecast.
-        label : string
-            A label (prefix) that is used for the forecast
-        output_file : str or None, default None
-            Output path for the plot file. If None, plot is not saved to file.
-        args :
-            Other arguments are passed to main plot() call
-        kwargs :
-            Other keyword arguments are passed to main plot() call
-        """
+        By default the `0.5` and `0.9` prediction intervals are plotted. Other
+        intervals can be choosen by setting `intervals`.
 
-        # matplotlib==2.0.* gives errors in Brazil builds and has to be
-        # imported locally
+        This plots to the current axes object (via ``plt.gca()``), or to ``ax``
+        if provided. Similarly, the color is using matplotlibs internal color
+        cycle, if no explicit ``color`` is set.
+
+        One can set ``name`` to use it as the ``label`` for the median
+        forecast. Intervals are not labeled, unless ``show_label`` is set to
+        ``True``.
+        """
         import matplotlib.pyplot as plt
 
-        label_prefix = "" if label is None else label + "-"
+        # Get current axes (gca), if not provided explicitly.
+        ax = maybe.unwrap_or_else(ax, plt.gca)
 
-        for c in prediction_intervals:
-            assert 0.0 <= c <= 100.0
+        # If no color is provided, we use matplotlib's internal color cycle.
+        # Note: This is an internal API and might change in the future.
+        color = maybe.unwrap_or_else(
+            color, lambda: next(ax._get_lines.prop_cycler)["color"]
```

Comment:
Perhaps we can use the CN notation for getting the default color?
https://stackoverflow.com/questions/42086276/get-default-line-colour-cycle

Reply:
We want to get the next colour in the cycle. In the above example, if `"black"` was not set explicitly, the true values would be blue, and the predictions would start with orange.